### PR TITLE
add two events to notification center

### PR DESCRIPTION
**Pull request by barshimi**
Close: #7539 

### Summary
📊🗨️🎈

Added a notification center feature to the client app that displays messages from Amplication in the workspace header. The feature uses React hooks, references, and analytics to track user interactions with the notification center and the messages.

> _`AnalyticsEventNames`_
> _Popovers show messages_
> _Autumn of feedback_

### Walkthrough
*  Add analytics events for notification center interactions ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-917333793bfc12bbce1fed70cf28308c74cf550fb42d3653f076a65ad209e697R154-R157), [link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L106-R116), [link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24R165-R173))
  - Define new event names in `analytics-events.types.ts` ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-917333793bfc12bbce1fed70cf28308c74cf550fb42d3653f076a65ad209e697R154-R157))
  - Track when a notification message is clicked in `WorkspaceHeader.tsx` ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L106-R116))
  - Track when the notification bell is clicked and toggle the popover state in `WorkspaceHeader.tsx` ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24R165-R173))
*  Refactor notification bell component and state in `WorkspaceHeader.tsx` ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L21-R27), [link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L86-R96), [link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L301-R322))
  - Import `useRef` hook and create a reference to the bell component ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L21-R27))
  - Comment out unused `versionAlert` state and replace it with `novuCenterState` ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L86-R96))
  - Wrap the bell component in a div with an onClick handler and pass the reference to the popover ([link](https://github.com/amplication/amplication/pull/7606/files?diff=unified&w=0#diff-7fdb758d72b986ee99ae091fcd9f9b1312ecd063170e9dcbd9d99d5f47812d24L301-R322))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 1]
```diff
@@ -83,11 +89,11 @@ const WorkspaceHeader: React.FC = () => {
   const history = useHistory();
   const { stigg } = useStiggContext();
   const { trackEvent } = useTracking();
-
+  const novuBellRef = useRef(null);
   const breadcrumbsContext = useContext(BreadcrumbsContext);
 
-  const [versionAlert, setVersionAlert] = useState(false);
-
+  // const [versionAlert, setVersionAlert] = useState(false);
```
Comment (mulygottlieb) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:95]: Perhaps remove the commented out line if no longer in use?

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 2]
```diff
@@ -103,7 +109,11 @@ const WorkspaceHeader: React.FC = () => {
   }, [history, apolloClient]);
 
   const onNotificationClick = useCallback((message: IMessage) => {
-    // your logic to handle the notification click
+    trackEvent({
+      eventName: AnalyticsEventNames.ClickNotificationMessage,
+      messageType: "now-buildSucceeded",
```
Comment (mulygottlieb) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:114]: did you mean "new-buildSucceeded"? Or perhaps I don't understand the context for "now" here?

**Review by mulygottlieb — APPROVED** [review]
Approving, see my minor comments for your consideration

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx (hunk 2 above)
Comment (barshimi) [reply packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:114]: it came from paz

**Pull request merged**